webapp/js/home.js

```diff
@@ -31,28 +31,28 @@ $(document).ready(function(){
     }
```

Comment:
This shouldn't be here, please merge master branch into your personal branch and then, merge your personal branch into this branch to solve this

Reply:
Fixed